### ISSUE TITLE
[Reclaim] Join meeting not joining current meeting in reclaim

### ISSUE THREAD

**Issue by hoangbnc** [raycast/extensions#10374]
### Extension

https://www.raycast.com/reclaim-ai/reclaim-ai

### Description

When going to Reclaim extensions "My Calendar", I can see the next event is a meeting and use "Join meeting" action:
<img width="754" alt="image" src="https://github.com/raycast/extensions/assets/99745909/aa4dc370-b010-47ea-8764-93f230250ff5">

However, the "Join (Current) Meeting" extension command would say "No meetings found".

Looking at the source code, it appears the command is looking for events with `moment/next`, which looks for events in calendars connected with Raycast, rather than looking for events in Reclaim. This seems weird to me considering this is Reclaim's extension, and there is a Reclaim "My Calendar" command

### Steps To Reproduce

1. Have a meeting event in Reclaim scheduled for "now"
2. Run "Join Meeting" command

**Comment by ajohnson1200** [comment]
Thanks for submitting this! I'm not able to reproduce this with my calendar but that doesn't mean there's not a bug.  Could you send me the email address you're using with Reclaim along with the day / time of the event that you were trying to join so that I can look up the event metadata?  My email address is aj @ reclaim.ai.

Thanks!

**Comment by ajohnson1200** [comment]
ah, I got your email, thanks!  I think I see what you're saying... it's not the My Calendar that's not working, it's the Join Meeting command that's at the top level:

![Screenshot 2024-01-30 at 10 24 50 AM](https://github.com/raycast/extensions/assets/683294/d175e737-727f-4091-a6c4-a2bc36da5dd8)

and that one isn't working for you.... yep, that's using the Next stuff, you're right... I'll get an internal bug filed for this and will let you know when we've got it fixed.

AJ

**Comment by hoangbnc** [comment]
@ajohnson1200 I see you guys decided to just remove the [command](https://github.com/raycast/extensions/pull/10486) 😭

**Comment by ajohnson1200** [comment]
yeah, we talked about it internally and came to the conclusion that a part of the problem is that "Join Meeting" by itself would likely always be semi-indeterminate and that a better user experience would simply be to make it explicit as to what meeting you were joining by having folks go through My Calendar.  You can get a pretty similar experience by doing something like this:

1) set up a hotkey for My Calendar, I've got mine mapped to ⌘ J like this:

![Screenshot 2024-02-06 at 3 31 07 PM](https://github.com/raycast/extensions/assets/683294/b61e3fa8-9a4e-44a2-bc44-22d4c55265b7)

2) whenever you want to join a meeting, use your hot key, and then up or down arrow to the right meeting (if it's not already selected), and ...

3) hit Enter... and voila, meeting is launched and you're 100% positive every time that you're joining the right meeting.

Give that a shot and let us know if that workflow works for you... I've been using it with the hot key combination for the last week and it's been really nice.

**Comment by hoangbnc** [comment]
I see. Thanks for the suggestion!
